test/models/album_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'test_helper'
+
+class AlbumTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+  test "an album must have a title, description and recorded_by" do
+    assert albums(:suburbs).valid?
+    albums(:suburbs).title = nil
+    assert_not albums(:suburbs).valid?
+    albums(:suburbs).description = nil
+    assert_not albums(:suburbs).valid?
```

Comment:
We should split each of these `valid?` checks into a separate test which contains essentially lines 8 through 10 (but changes the specific attribute being set to nil).

The reason for isolating our tests that way is because each `assert_not` after the first one is not beginning from the same starting point.

Even if we didn't have `validates :description, presence: true` in our `Album` class this test would pass -- because `albums(:suburbs)` is already invalid due to having no title. When we split this test up into three separate tests, we can be assured that the fixture for the album we're testing starts off exactly as specified in the fixture file each time.